### DISCUSSION TITLE
Coverage map on uneven surface

### DISCUSSION THREAD

**Discussion by ListenToAJ** [NVlabs/sionna#668] [General]
Hello, I am new to Sionna and coverage maps.  My goal is to have  a coverage map of a natural landscape, I have created a blender xml output and loaded the scene as well as transmitter and reciever.  I want to generate a coverage map that goes along with the landscape, (For example if the transmitter is in a low crater signal outside the crater would be limited).  How can I create a coverage map that will lay on top of the uneven landscape I have loaded in?

**Comment by merlinND** [comment]
Closing as duplicate of https://github.com/NVlabs/sionna/discussions/610.
